frontend/src/pushapi.js

```diff
@@ -102,92 +51,66 @@ angular.module('clouway-push', [])
       /**
        * Open channel using specified channel token.
        *
-       * @param {String} channelToken the token to use for opening channel.
+       * @param {String} data the data from server to use for opening channel.
        */
-      var openChannel = function (channelToken) {
-        var channel = new goog.appengine.Channel(channelToken);
-        var socket = channel.open();
-
-        socket.onmessage = onMessage;
-
-        socket.onerror = function (errorMessage) {
-          establishConnection();
-        };
+      service.openChannel = function (data) {
```

Comment:
This function should remain anonymous, not exposed in the service.

Reply:
You are right. I will do